src/components/resources-section/average/Average.module.scss

```diff
@@ -0,0 +1,4 @@
+.rating_number {
+  font-size: x-large;
```

Comment:
Try to use rem or em, this is a better way to work with responsive design.

Reply:
Very good advise! Already corrected.